docs/LogGuide.md

```diff
@@ -1972,21 +2176,30 @@ Parsed Log Line Examples:
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
-It seems likely that this line was added in order to extend functionality
-for the [NetworkBuff](#line26),
-[NetworkBuffRemove](#line30),
-and [NetworkActionSync](#line37)
-log lines without breaking previous content or plugins.
+This line conveys all current status effects on an entity.
+This can be useful if a plugin or overlay was started after zoning in.
+
+#### Data Fields
+
+This is a variable-length line.
+It can expand up to 30 status effects.
+Beginning with the field called `data3`, each status effect takes three fields.
+
+The first data field for each Trio is some unknown data plus the effect ID in the latter 2 bytes.
+i.e. `field && 0xffff` will get you the effect ID.
+
+The second is the remaining duration as a 32-bit float.
+It comes in as a hex integer, so you will need to do the conversion to float manually.
```

Comment:
This maybe should be clarified as something like "a binary-encoded float" or "a raw byte representation of a float", as the current language could be read as this being correct when it's not:
```cs
Float duration = (Float) Int32.Parse(durationString);
```

Reply:
Do you happen to have a C# example of how it would be done correctly?